src/Components/Counter.jsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Display from './Display';
 import ButtonPad from './ButtonPad';
 
-
 function Counter() {
-    return (
-        <div className="counter">
-            <Display/>
-            <ButtonPad/>
-        </div>
-    )
+  const [count, setCount] = useState(0);
+  const handleReset = () => setCount(0);
+  const changeCount = (delta) => setCount(count + delta);
+
+  return (
+    <div
+      className='counter'
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        fontSize: 'calc(16px + 6vmin)',
+        paddingTop: '5px',
```

Comment:
wondering what the reasoning was to put your styles inline here.  Sometimes inline styling causes a little bit of difficulty tracking down and modifying later, if using an external style sheet.